### DISCUSSION TITLE
v0.0.6

### DISCUSSION THREAD

**Discussion by aniruddha-adhikary** [aniruddha-adhikary/mrt-buddy#19] [Announcements]
- friendlier instruction message for those who are holding the card incorrectly.

==

আপডেট আসলেই নোটিফিকেশন পেতে চান? প্লে স্টোর এ অনুমোদন পাওয়ার আগ পর্যন্ত এই WhatsApp চ্যানেলেই সব আপডেট জানানো হবে!

https://whatsapp.com/channel/0029VaurMehLI8Yeb3STq42g

<hr /><em>This discussion was created from the release <a href='https://github.com/aniruddha-adhikary/mrt-buddy/releases/tag/v0.0.6'>v0.0.6</a>.</em>

**Comment by scobie-dobie-do** [comment]
Great Job bhai!!! Though my device doesn't support nfc ToT

**Reply by aniruddha-adhikary** [reply]
Thank you so much for the support!

**Comment by radifulhoque258** [comment]
Is there any way to use it in ios?

**Reply by aniruddha-adhikary** [reply]
We haven't started building the app for iOS because the market for iOS is really small in Bangladesh. But if anyone wants to start more then welcome to!

**Reply by AbdalMahfuz** [reply]
@aniruddha-adhikary  I want to build the iOS app. 
Requesting your attention.

**Reply by aniruddha-adhikary** [reply]
@AbdalMahfuz I am thinking for feature-parity sake, we should convert this project to Flutter. Does that work?

**Reply by mohd-asfaq-e-azam-rifat** [reply]
Hi @aniruddha-adhikary
I'm a Software Engineer as well and interested to contribute in this project. If you're planning to shift to Flutter, count me in. I appreciate your idea.

**Comment by AbdalMahfuz** [comment]
@aniruddha-adhikary  I can contribute in Native iOS.
